tests/app/test_jinja_filters.py

```diff
@@ -139,7 +139,7 @@ def test_format_unit_welsh(self):
         self.assertEqual(format_unit("duration-year", 100, "long"), "100 mlynedd")
 
     @patch("app.jinja_filters.flask_babel.get_locale", Mock(return_value="en_GB"))
-    def test_format_unit_input_label(self):
+    def test_format_unit_input_label_short(self):
```

Comment:
We could add a functional test using something we already used in our tests, look through html since the tooltip element is in body of the document.

Reply:
Had to update the labels in the test schema in order for the functional test to work correctly.